RyanNoelk/OpenEats
# Feature/polyfill

Adds a wrapper for using poly fills. The goal being to only load them if the browser doesn't support all the feature OpenEats uses.

Fixes #197 

To Test: 
- Load the site and make sure no polyfills load.
- Load the site where we need polyfills and make sure the site loads and the poly fills loads.
